src/mumble/LookConfig.ui

```diff
@@ -636,6 +636,9 @@
         <property name="toolTip">
          <string>A user that is silent for the given amount of seconds will be removed from the Talkin UI.</string>
         </property>
+        <property name="minimum">
+         <number>-1</number>
```

Comment:
If I understand the code correctly, ``-1`` will be displayed when your new option takes effect, right? I think that is not very nice from a user perspective.
It would be better, if a string was displayed like "Disabled" or something similar. Alternatively, you could create a check-box to enable/disable this feature and make the slider inactive, if it is disabled.

Reply:
Yes, you understood it perfectly.
I thought about using a checkbox as well, but then there are some UI things to consider and I'm not really good at that.
E.g. if I add a checkbox to all the checkboxes in the UI above, it will influence the slider which is much further down in the UI and there is no visible indication that they belong together.

If I instead add the checkbox directly to the option it looks strange to have a checkbox out of nowhere, and the question is how indentation should be done:
![grafik](https://github.com/mumble-voip/mumble/assets/13928225/8337b8f0-672d-4724-a0fb-711ee79e6982)

Reply:
Maybe place a checkbox to the right of a label that says "Remove silent user". And add the number box to a new row with the label "after...". Then also add the Qt suffix "seconds" to the number box.

But disregarding what we choose here: everything is better than just "-1"

Reply:
We could do it inside a groupbox but that would destroy the table like layout:
![grafik](https://github.com/mumble-voip/mumble/assets/13928225/62efe5e3-5c80-491b-b797-6eecfc08ef2a)

I could do it with two separate rows, and indent the "after" label manually, but I do not think it is that nice to have one and the same thing span two rows. Any idea?

Reply:
What if you place the checkbox to the right of the label instead of to the left. So the checkbox is horizontally aligned with the number boxes? Maybe the "Audio Input" "Misc" section inspires you. There it is also handled slightly differently.

Reply:
I can't really place it on the right using a groupbox(and tbh I would personally not like the look of that).
Instead I just did it analogous to the "Abbreviate channel names":
![grafik](https://github.com/mumble-voip/mumble/assets/13928225/766572bc-8149-4ca9-ab47-d941bcbbafa5)

Do you think that would be acceptable?

Reply:
Yep that is neat. :+1: